github-overrides.json

```diff
@@ -0,0 +1,4 @@
+{
+  "CloudCV": "Cloud-CV",
+  "Liquid Galaxy project": "LiquidGalaxyLAB"
```

Comment:
This value is also in the JSON, and can be extracted as part of the fallback strategy to find the correct exact match.